plugins/generator-1.18.2/forge-1.18.2/utils/procedures.java.ftl

```diff
@@ -57,6 +57,14 @@
     </#if>
 </#macro>
 
+<#macro procedureOBJToLogicCode object="">

```

Comment:
You can reuse procedureOBJToConditionCode for this purpose, just add another optional argument defaultValue=true, that you set to false for your cases.

Same for 1.19.2